### DISCUSSION TITLE
ADC_TEMPERATUIRE  really ???

### DISCUSSION THREAD

**Discussion by Yardie-** [SpenceKonde/megaTinyCore#390] [Q&A] [answered]
I love this direct read of internals but..
ADC_TEMPERATUIRE
on the V2 parts 
Whos idea was that ??? 
Spence are you getting even more creative?

**Comment by SpenceKonde** [comment, marked as answer]
This will be corrected in the next release (Also, I thought I had fixed this one) 

Codespell sometimes leaves something to be desired. It's pure typo from having recently switched to an external keyboard after my laptop (on it's 4th keyboard - it has in the past year started destrotying keyboards after only a few months) keyboard started to give out again. I still haven't trained my motor neurons well enough on the external keyboard. 

In the future, if it is something that is clearly a defect in the core that can go straight to an issue. The chance that I am defining a new word to describe what the ADC is measuring is unlikely (then again, I haven;'t checked whether that channel accurately reflects temperature - maybe  it will turn out to be so bad that it does need a different word to describe it!

**Comment by Yardie-** [comment]
I only saw it in the documentation I am not sure if it's the same in the code.
Haha I thought as much. 
We are all, after all, only human.
Spence I love your work you have opened so many doors for me.
Thank you.
